### DISCUSSION TITLE
Authentication AdonisJs5

### DISCUSSION THREAD

**Discussion by Mykhalchuk19** [adonisjs/core#1393] [Help] [answered]
I installed @adonisjs/auth package and ran ace command. Then I tried to run migration but I got an error "Exception: Invalid "auth" config. Missing value for "guard". Make sure to set it inside "config/auth".  I use mysql database and I want to use Jwt guard. I tried to set value for "guard", but it didn't help me. Maybe I can't understand anything?

**Comment by RomainLanz** [comment, marked as answer]
Hey @Mykhalchuk19! 👋 

There's no JWT Guard in AdonisJS 5.
Also, ensure to correctly select the guard **with space** when running the `node ace invoke` command.
